test/stress/emf/parameters.yml

```diff
@@ -0,0 +1,72 @@
+# Receivers that agent needs to tests
+receivers: ["emf"]
+
+#Test case name
+test_case: "emf_stress"
+validate_type: "stress"
+# Only support metrics/traces/logs
+data_type: "metrics"
+# Number of metrics to be sent or number of log lines being written  each minute
+values_per_minute: "<values_per_minute>"
+# Number of seconds the agent should run and collect the metrics. In this case, 5 minutes
+agent_collection_period: 300 
+
+cloudwatch_agent_config: "<cloudwatch_agent_config>"
+
+# Metric that the test needs to validate; moreover, the stress validation already has
+# InstanceID dimension; therefore, does not need to validate it
+# https://github.com/aws/amazon-cloudwatch-agent-test/pull/109/files#diff-47c87373e751dd9fd5ce504e44b320765c8b84d6cde524a4e8a32cfa34674165R124-R135
+metric_namespace: "CloudWatchAgentStress"
+metric_validation: 
+  - metric_name: "procstat_cpu_usage"
+    metric_sample_count: 300
```

Comment:
The metric sample count is 300 because the validator run in 300 s and[ every second its collects 1 metrics ](https://github.com/aws/amazon-cloudwatch-agent-test/pull/174/files#diff-74c52ae0b9939a3ad9e79606cd428ea97f70edfb11f46d8dd08e1dd9176016d8R29) (depends on the OS)